validate.sh

```diff
@@ -27,11 +27,11 @@ GOGLOB="${GOGLOB/ docs/}"
 GOGLOB="${GOGLOB/ vendor/}"
 
 # Check that there are no formatting issues
-GOFMT_LINES=`gofmt -s -l $GOGLOB | wc -l | xargs`
+GOFMT_LINES=`gofmt -s -l $GOGLOB | tr '\\\\' '/' | wc -l | xargs`
```

Comment:
Why did you need to change this?

Reply:
go tooling is not shell aware, so when you use git bash to run validate.sh, it call gofmt, and gofmt returns paths\like\this, which is incompatible with bash, so this reformats them as paths/like/this et voila.